src/osd/OSD.h

```diff
@@ -1582,9 +1582,9 @@ class OSD : public Dispatcher,
 
 private:
   /**
-   *  @defgroup monc helpers
-   *
-   *  Right now we only have the one
+   * \defgroup monc helpers
```

Comment:
it's javadoc style, see https://www.stack.nl/%7Edimitri/doxygen/manual/docblocks.html#docexamples .